packages/components/checkbox/src/checkbox.ts

```diff
@@ -106,13 +103,18 @@ const Checkbox = defineComponent({
     const hasConsumerStoppedPropagationRef = ref(false)
 
     const modelValue = useModel(props, 'modelValue')
+    const proxyChecked = computed({
+      get: () => modelValue.value !== undefined ? modelValue.value : checkedProp.value !== undefined ? checkedProp.value : undefined,
+      set: () => {
```

Comment:
This required?

Reply:
If you can solve it another way, you can pass the tests without changing them.